Formula/tea.rb

```diff
@@ -0,0 +1,32 @@
+class Tea < Formula
+  desc "Command-line tool to interact with Gitea servers"
+  homepage "https://gitea.com/gitea/tea"
+  url "https://gitea.com/gitea/tea/archive/v0.8.0.tar.gz"
+  sha256 "6c73c0a7b66cdfd1e5a302257d54df06a3a41eb9bdbfeb547966db431ae23b23"
+  license "MIT"
+  head "https://gitea.com/gitea/tea.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}")
+    bash_completion.install "contrib/autocomplete.sh" => "tea"
+    zsh_completion.install "contrib/autocomplete.zsh" => "_tea"
+
+    system "#{bin}/tea", "shellcompletion", "fish"
```

Comment:
Is there a way to specify where it outputs? 
```suggestion
    system bin/tea", "shellcompletion", "fish"
```

Reply:
Unfortunately, no.